tools/cleanup/cleanup.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a script helping clean up stale resources
+
+source $(dirname ${BASH_SOURCE})/../../scripts/library.sh
```

Comment:
Since this is not intended to be sourced

```suggestion
source $(dirname $0)/../scripts/library.sh
```